src/irr/video/COpenGLDescriptorSet.h

```diff
@@ -34,6 +35,7 @@ class COpenGLDescriptorSet : public IGPUDescriptorSet
         {
             const GLuint* textures;
             const GLenum* targets;//for when ARB_multi_bind isn't there
+            const GLuint* samplers;
         };
         struct SMultibindTextureImages
         {
```

Comment:
does IGPUDescriptorSet hold onto the interface object refcounted pointers that provide these values?

Reply:
yes, resolved